app/uk/gov/hmrc/economiccrimelevyaccount/connectors/DesConnector.scala

```diff
@@ -54,7 +49,8 @@ class DesConnector @Inject() (
     retryFor[ObligationData]("DES - obligation data")(retryCondition) {
       httpClient
         .get(url"${desUrl(eclRegistrationReference.value)}")
-        .transform(_.addHttpHeaders(desHeaders: _*))
+        .setHeader((HeaderNames.authorisation, s"Bearer ${appConfig.desBearerToken}"))
+        .setHeader((CustomHeaderNames.environment, appConfig.desEnvironment))
```

Comment:
Why did you change transform with `desHeaders` method into separate calls to set headers ?

Reply:
So that we could test that they are getting sent out, it wasn't possible with transform